build.sbt

```diff
@@ -266,12 +304,26 @@ val mtagsSettings = List(
   },
   crossTarget := target.value / s"scala-${scalaVersion.value}",
   crossVersion := CrossVersion.full,
-  Compile / unmanagedSourceDirectories ++= multiScalaDirectories(
-    (ThisBuild / baseDirectory).value / "mtags",
+  Compile / unmanagedSourceDirectories := Seq(
+    (ThisBuild / baseDirectory).value / "mtags" / "src" / "main" / "scala"
+  ),
+  Compile / unmanagedSourceDirectories ++= crossScalaStablePC(
     scalaVersion.value,
+    ifStablePC = Nil,
+    default = multiScalaDirectories(
+      (ThisBuild / baseDirectory).value / "mtags",
+      scalaVersion.value,
+    ),
   ),
   // @note needed to deal with issues with dottyDoc
   Compile / doc / sources := Seq.empty,
+  libraryDependencies ++= crossScalaStablePC(
+    scalaVersion.value,
+    ifStablePC = List(
+      "org.scala-lang" %% "scala3-presentation-compiler" % scalaVersion.value
```

Comment:
Are we sure this will be the artifact? Maybe we could add these changes later?

Reply:
I reverted those changes. I'll add them when PR for dotty is created and ready.